deepchem/feat/molecule_featurizers/grover_featurizer.py

```diff
@@ -118,26 +118,14 @@ def _make_mol_graph(self, mol: RDKitMol) -> GraphData:
         from deepchem.feat.molecule_featurizers.dmpnn_featurizer import bond_features
         smiles = Chem.MolToSmiles(mol)
         n_atoms = mol.GetNumAtoms()  # number of atoms
-        n_bonds = 0  # number of bonds
         f_atoms = []  # mapping from atom index to atom features
         f_bonds = [
         ]  # mapping from bond index to concat(from_atom, bond) features
-        a2b: List[List[int]] = [
```

Comment:
Why are we deleting these?

Reply:
The idea behind these changes is to extract `a2b` and other attributes from the graph using edge index. This can be done by the newly added `extract_grover_attributes` method. So, the attributes extracted from featurizer are not needed.